src/hotspot/cpu/ppc/globals_ppc.hpp

```diff
@@ -60,7 +60,7 @@ define_pd_global(bool,  VMContinuations, true);
 
 // Use large code-entry alignment.
 define_pd_global(uintx, CodeCacheSegmentSize,  128);
```

Comment:
Shouldn't also changing CodeCacheSegmentSize to match CodeEntryAlignment save a little more in the code cache?

If we did all alignment on the actual address rather than an offset that is relative, I think we could get rid of some of the constraints that require CodeCacheSegmentSize to be at least as large as CodeEntryAlignment and OptoLoopAlignment.

Reply:
This may make sense. However, almost all platforms use a CodeCacheSegmentSize of 128 Bytes. So, I think this should better get addressed separately for all platforms.

Reply:
Interesting suggestion. I created JDK-8326571 as follow-up to work on the topic.